Skims/plugins/TrackIsolationFilter.cc

```diff
@@ -127,8 +127,10 @@ bool TrackIsolationFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSe
 
   if( vertices->size() > 0) {
 
-     for( pat::PackedCandidateCollection::const_iterator pf_it = pfCandidatesHandle->begin(); pf_it != pfCandidatesHandle->end(); pf_it++ ) {
-
+    // for( pat::PackedCandidateCollection::const_iterator pf_it = pfCandidatesHandle->begin(); pf_it != pfCandidatesHandle->end(); pf_it++ ) {
+for(size_t i=0; i<pfCandidatesHandle->size();i++)
+    {
```

Comment:
Maybe you can switch back to the original const_iterator after you change edm::View to std::vector